libprecompiled/ChainGovernancePrecompiled.cpp

```diff
@@ -357,6 +388,23 @@ int ChainGovernancePrecompiled::grantOperator(
                                  " can't grantOperator operator " + _userAddress));
         return result;
     }
+    // add permission of SYS_CNS
```

Comment:
It is recommended to abstract the logic of add permission to sys tables into function